### ISSUE TITLE
Modify `riskmodel.mental`,  `riskmodel.physical`,  `riskmodel.sexual` fields from question type select to string

### ISSUE THREAD

**Issue by jayesse** [OpenFn/unicef-thailand#93]
## Background, context, and business value

Update the `Flow 2 - 2`  upsert job by updating the question type of [these three fields](https://docs.google.com/spreadsheets/d/1f1fT3qmM4mKT98AaJ0ArlgONQRC-W9ghoa-j4BswwbM/edit#gid=1877091315&range=H177:H179) from `select` to `string`

## The specific request, in as few words as possible
 The following fields are currently written as select values in [L2986 - L3009](https://github.com/OpenFn/primero-thailand/blob/staging/jobs/1b-upsertCases.js#L2986-L3009) of the upsert job script.
 
```
mental_37b1218
physical_dd0399d
sexual_b54b18b
```

 The client has requested that we change these to text fields that would accept string values from Child Shield. The Primero system has already been modified to accept string input.

### expression.js
*Please make changes in the `staging` branch!* ⚠️
Link to the job itself in Github: https://github.com/OpenFn/primero-thailand/blob/staging/jobs/1b-upsertCases.js

#### adaptor
`language-primero@2.9.1`

### state.json
This is step 2 job in a flow. 
1. Run job1 `/staging/jobs/1-getPatientData.js` to generate state for job 2. 
- state.configuration: `MOPH Thailand HIS API - DEMO Alpha (Unicef)`
- state.data: Use this Message: https://openfn.org/projects/primero-thailand-io-unicef---staging/runs/06361e9e-8831-748d-95f4-2cc588d3cf3a
2.  Use the final state of the first job to run the 2nd job
- `state.configuration`: see LP `Primero Thailand Demo API User (Open Function)` (Note... this user is for **basic auth**.)

## output.json
There following fields should accept string values from Child Shield.  Verify this by re-running the Flow 2-1 job with [this message](https://www.openfn.org/projects/primero-thailand-io-unicef---staging/messages/0637c24a-ec8e-7fe6-95a3-04c970b9758c), which will then trigger a re-run of the Flow 2-2 job. Afterwards, check the Flow 2-2 logs for the following fields:

```
mental_37b1218
physical_dd0399d
sexual_b54b18b
```

## Toggl 

UNICEF Thailand

**Comment by mtuchi** [comment]
@aleksa-krolls i fixed this  [63fa80f](https://github.com/OpenFn/primero-thailand/commit/63fa80f3afd57c7b0275ef73b764b6c9541eb6cd), Please test on `platform-app` cc @jayesse

**Comment by jayesse** [comment]
@mtuchi Your latest update fixed the issue, as evident from [this log](https://www.openfn.org/projects/primero-thailand-io-unicef---staging/runs/063861d8-5485-7e72-83bf-a87c234e7c3b). 

The values are also correctly displayed in Primero. Thus I'm closing the issue. Thanks for your assistance.
